### PR TITLE
fetch from internet and filter per boundable

### PR DESCRIPTION
**Pull request by enriquesouza**
Now we are fetching the assets from internet and filtering them by boundables. So It should be 100% dynamic.

image

image

### CODE REVIEW

File: VultisigApp/VultisigApp/Model/TransactionMemo/TransactionMemoBondMayaChain.swift [hunk 1]
```diff
@@ -40,30 +41,45 @@ class TransactionMemoBondMayaChain: TransactionMemoAddressable, ObservableObject
             }
         }
     }
-    
+
     private var cancellables = Set<AnyCancellable>()
-    
-    required init(assets: [String]) {
+
+    let id = UUID()
```
Comment (johnnyluo) [inline VultisigApp/VultisigApp/Model/TransactionMemo/TransactionMemoBondMayaChain.swift:47]: this one doesn't used any where , could you please remove it

File: VultisigApp/VultisigApp/Model/TransactionMemo/TransactionMemoUnbondMayaChain.swift [hunk 2]
```diff
@@ -34,30 +37,44 @@ class TransactionMemoUnbondMayaChain: TransactionMemoAddressable, ObservableObje
             }
         }
     }
-    
+
     private var cancellables = Set<AnyCancellable>()
-    
-    required init(assets: [String]) {
+
+    let id = UUID()
```
Comment (johnnyluo) [inline VultisigApp/VultisigApp/Model/TransactionMemo/TransactionMemoUnbondMayaChain.swift:43]: this one is not used anywhere either, could you please remove it?

**Review by johnnyluo — APPROVED** [review]
LGTM

File: VultisigApp/VultisigApp/Model/TransactionMemo/TransactionMemoBondMayaChain.swift (hunk 1 above)
Comment (enriquesouza) [reply VultisigApp/VultisigApp/Model/TransactionMemo/TransactionMemoBondMayaChain.swift]: removed

File: VultisigApp/VultisigApp/Model/TransactionMemo/TransactionMemoUnbondMayaChain.swift (hunk 2 above)
Comment (enriquesouza) [reply VultisigApp/VultisigApp/Model/TransactionMemo/TransactionMemoUnbondMayaChain.swift]: Removed

**Pull request merged**